### ISSUE TITLE
Missing characters

### ISSUE THREAD

**Issue by rjwut** [SpeedyOrc-C/HoYo-Glyphs#22]
There are some characters available for Genshin fonts that are missing:

* **Fontaine**
  * Letters `K`, `Q`, `Y`, `Z`: Glyphs available at the [Fontaine Script article on the Genshin Impact Fandom Wiki](https://genshin-impact.fandom.com/wiki/Fontaine_Script)
  * Digits `0`-`9`: Displayed by the ["Fateometer" gadget](https://genshin-impact.fandom.com/wiki/%22Fateometer%22)
* **Sumeru**
  * Letter `X`: Glyph available at the [Sumeru Script article on the wiki](https://genshin-impact.fandom.com/wiki/Sumeru_Script)
  * Digits `1`-`5`: Lyney's charged attack has up to 5 stacks of Prop Surplus, which is displayed in the game in this font. Lisa's attacks also apply up to 3 Conductive stacks, displaying the digit in the game.

**Comment by SpeedyOrc-C** [comment]
Thanks for providing the information

**Comment by SpeedyOrc-C** [comment]
Now drawing Fontaine numerals...

**Comment by rjwut** [comment]
I should also mention that there's an alternate character in [Inazuman script](https://genshin-impact.fandom.com/wiki/Inazuman_Script) that appears to be a possible alternate glyph for `S` or `Z`.

**Comment by SpeedyOrc-C** [comment]
Shall we map that character to colon `:` or double s `ß` ?

**Comment by rjwut** [comment]
My personal inclination is to map it to `:`, but I'll defer to your judgement.

**Comment by ninjamask** [comment]
During the fight of local legend "Tlatzacuilotl" you can see Sumeru Script numbers 1-6. eg https://youtu.be/SznZ3KWFEO4

**Comment by SpeedyOrc-C** [comment]
> During the fight of local legend "Tlatzacuilotl" you can see Sumeru Script numbers 1-6. eg https://youtu.be/SznZ3KWFEO4

I think we have already seen numbers 1~3 from Lisa, interesting.

**Comment by SpeedyOrc-C** [comment]
@rjwut X for Sumeru script added

**Comment by EnigmaticDreamMachine** [comment]
The `J` for `ZZZ-System` is missing.
Here's an [image](https://static.wikia.nocookie.net/zenless-zone-zero/images/7/78/ZZZ_Special.png/revision/latest/scale-to-width-down/1000?cb=20240803190633) pulled from the [wiki page](https://zenless-zone-zero.fandom.com/wiki/Language) for reference.

**Comment by SpeedyOrc-C** [comment]
@EnigmaticDreamMachine Letter J added

**Comment by Bellimorko** [comment]
Yeah, missing characters can really break the flow of the story and gameplay in any update. It’s always strange when you get used to a team setup and suddenly someone’s gone or unavailable. I’ve had a few moments like that while exploring with my favorite Genshin Impact Characters — you can check more about them here www.genshinimpactcharacters.com — it throws off the balance and strategy completely. I usually adjust my lineup to fill the gap, but it never feels quite the same until they return.

**Comment by 98ColdDew** [comment]
截止最新的2025.05.29版本，须弥文的X、枫丹文的KQYZ，还是没有补全（
Sumeru letter X and Fontaine letters KQYZ are still missing in Ver 2025.05.29.
